e2e/test/scenarios/sharing/public-sharing-embed-button-behavior.cy.spec.js

```diff
@@ -194,7 +194,7 @@ describe("embed modal display", () => {
   });
 
   describeEE("when the user has a paid instance", () => {
-    it("should display a link to the Interactive embedding settings", () => {
+    it("should display a disabled state and a link to the Interactive embedding settings", () => {
```

Comment:
[test this case](https://www.notion.so/metabase/Streamline-embed-flow-when-embedding-features-are-off-13869354c90180b4bc02ca29ebf61ad2?pvs=4#14c69354c9018075b200f95110f8f407): disabled